apps/coscrad-frontend/src/components/higher-order-components/carousel.tsx

```diff
@@ -30,25 +32,18 @@ export const Carousel = <T,>({ propsForItems, Presenter }: CarouselProps<T>) =>
     const propsForSelectedItem = propsForItems[indexToUse];
 
     return (
-        <Card>
+        <VocabularyListWrapper>
```

Comment:
Definitely the wrong place for this. The Carousel is reused for other features besides the vocabulary lists.